src/gui/editor/Editor.js

```diff
@@ -103,17 +104,16 @@ export class Editor implements ImageHandler {
 				sanitizeToDOMFragment: this._sanitizer,
 			})
 			.addEventListener('keyup', (e) => {
-				// Resolve logins.getUserController dependenciy and uncomment block
-				// if (!this._tutanotaProperties.sendPlaintextOnly) {
-				if (e.which === 32) {
-					let blocks = []
-					squire.forEachBlock((block) => {
-						blocks.push(block)
-					})
-					createList(blocks, /^1\.\s$/, true) // create an ordered list if a line is started with '1. '
-					createList(blocks, /^\*\s$/, false) // create an ordered list if a line is started with '1. '
+				if (!logins.getUserController().props.sendPlaintextOnly) {
```

Comment:
Depending on globals in UI components is very bad, I think we should pass it in (maybe that's what comment meant, I'm not sure).